src/js/view/controls/controlbar.js

```diff
@@ -277,9 +277,9 @@ export default class Controlbar {
         _model.change('dvrLive', (model, dvrLive) => {
             const { liveBroadcast, notLive } = localization;
             const liveElement = this.elements.live.element();
-
-            toggleClass(liveElement, 'jw-dvr-live', !dvrLive);
-            setAttribute(liveElement, 'aria-label', dvrLive ? liveBroadcast : notLive);
+            const dvrLiveFalse = dvrLive === false;
```

Comment:
rename to `dvrNotLive`

Reply:
add a comment explaining that 'jw-dvr-live' actually means 'DVR mode but not at the live edge'.